apps/explorer/src/pages/transaction-result/Events.tsx

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Disclosure } from '@headlessui/react';
+import { ChevronRight12 } from '@mysten/icons';
+import {
+    formatAddress,
+    type SuiEvent,
+    type TransactionEvents,
+} from '@mysten/sui.js';
+import clsx from 'clsx';
+import { type ReactNode } from 'react';
+
+import { CopyToClipboard } from '~/ui/CopyToClipboard';
+import { Divider } from '~/ui/Divider';
+import { ObjectLink } from '~/ui/InternalLink';
+import { JsonViewer } from '~/ui/JsonViewer';
+import { Text } from '~/ui/Text';
+
+function formatTypeName(typeName: string) {
+    const split = typeName.split('<');
+    if (split.length <= 1) {
+        return split[0];
+    }
+
+    return `${split[0]}<...>`;
+}
+
+function formatType(type: string) {
+    const split = type.split('::');
+    const p0 = formatAddress(split[0]);
+    const p1 = split[1];
+    const p2 = formatTypeName(split.slice(2).join('::'));
+
+    return [p0, p1, p2].join('::');
+}
+
+function EventRow({ title, children }: { title: string; children: ReactNode }) {
+    return (
+        <div className="flex flex-col gap-2 md:flex-row md:gap-10">
+            <dt className="w-full flex-shrink-0 text-pBody font-medium text-steel-darker md:w-28">
+                {title}
+            </dt>
+            <dd className="ml-0 min-w-0 flex-1 break-words break-all leading-none">
+                {children}
+            </dd>
+        </div>
+    );
+}
+
+function Event({ event }: { event: SuiEvent }) {
```

Comment:
If we haven't already, can we delete the old code related to events?

Reply:
I believe they are gone, I was looking for it.